Gemfile

```diff
@@ -6,6 +6,5 @@ require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundle
 # Declare your gem's dependencies in topological_inventory.gemspec.
 # Bundler will treat runtime dependencies like base dependencies, and
 # development dependencies will be added by default to the :development group.
-gem 'rbac-api-client', :git => "https://github.com/RedHatInsights/insights-rbac-api-client-ruby", :branch => "master"
-
+gem 'rbac-api-client', :git => "https://github.com/mkanoor/insights-rbac-api-client-ruby", :branch => "remove_policy"
```

Comment:
I had to do this because I am waiting for this PR to get merged.
https://github.com/RedHatInsights/insights-rbac-api-client-ruby/pull/4